tests/longevity-in-memory-36GB-4days.yaml

```diff
@@ -1,12 +1,11 @@
 test_duration: 5760
-prepare_write_cmd: "cassandra-stress write cl=QUORUM n=36000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=500 -pop seq=1..36000000"
-stress_cmd: ["cassandra-stress mixed cl=QUORUM duration=5760m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -pop seq=1..36000000 -log interval=5",
-             "cassandra-stress counter_write cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1 -pop seq=1..1000000"
-            ]
-stress_read_cmd: ["cassandra-stress read cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -rate threads=20 -pop seq=1..36000000 -log interval=5",
-                  "cassandra-stress counter_read cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -rate threads=10 -pop seq=1..1000000"]
-n_db_nodes: 6
-n_loaders: 3
+prepare_write_cmd: ["cassandra-stress write         cl=QUORUM n=36000000     -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=500 -pop seq=1..36000000 -col 'size=FIXED(200) n=FIXED(5)' -log interval=10",
```

Comment:
why not -col 'size=FIXED(1024) n=FIXED(1)'?

Reply:
because you run pre-create schema with the default columns.
So, either we change that or we change it here.